kymatio/scattering1d/frontend/keras_frontend.py

```diff
@@ -7,15 +7,16 @@
 
 
 class ScatteringKeras1D(ScatteringKeras, ScatteringBase1D):
-    def __init__(self, J, Q=1, max_order=2, oversampling=0):
```

Comment:
Shouldn't this just be `Q` to allow tuples to pass through? Same with the other frontends.

Reply:
Missed frontends